django_inbound_email/backends/mailgun.py

```diff
@@ -65,11 +74,13 @@ def parse(self, request):
             cc=cc,
             bcc=bcc,
         )
+        if date:
+            email.extra_headers['date'] = date
         if html is not None and len(html) > 0:
             email.attach_alternative(html, "text/html")
 
         # TODO: this won't cope with big files - should really read in in chunks
-        for n, f in request.FILES.iteritems():
+        for n, f in request.FILES.items():
```

Comment:
It might be worth using `six.iteritems()` in these changes, as `items()` in Python 3.x and 2.x are not the same thing.